src/views/templates/list/hooks/useVirtualMachineTemplatesColumns.ts

```diff
@@ -19,7 +19,13 @@ const useVirtualMachineTemplatesColumns = (): TableColumn<K8sResourceCommon>[] =
       sort: 'metadata.namespace',
     },
     {
-      title: t('Boot source availability'),
+      title: t('Workload profile'),
+      id: 'workload',
+      transforms: [sortable],
+      sort: 'objects[0].spec.template.metadata.annotations.["vm.kubevirt.io/workload"]',
```

Comment:
hmm, here you assume only one object and that it is the vm, but elsewhere we use a method to get the vm object, ( can we can assume one object and that it is the vm in other places too ? )

Reply:
I am not sure what we can assume in other places. In which places... ? But if you wonder about the line 25 for sorting the workload profile, I haven't found any better way to define which info we want to sort by. If you have some another/better working solution, I am happy to see/try it. Thank you! :)